### DISCUSSION TITLE
Contributions for "other" renderers

### DISCUSSION THREAD

**Discussion by KingOfTac** [markdoc/markdoc#533] [Q&A] [answered]
Would contributions for renderers for other languages/frameworks be welcome in this repository, or would it be better for the creators of those renderers to maintain them separately in their own projects?

If other renderers are welcome in this repository, then what would that look like from a tooling perspective? This repository is currently geared towards frontend but what would it look like if someone wanted to contribute a renderer for Rust as an example?

I'm curious what the team's thoughts are on this. I have a renderer written in GO that I have been thinking of making public but wanted to pose the question here before I do.

**Comment by mfix-stripe** [comment, marked as answer]
@KingOfTac I think it makes sense to develop other renderers outside of this repository, and then incorporate them in if they fit the spirit of Markdoc. For example, maybe a plaintext renderer fits well within this repo, but a smalltalk renderer doesn't (just a random example I'm made up).
